### ISSUE TITLE
Package installs correctly, but then all pages display the error (presumably due to config rebuild as `project-config/rebuild` also throws same error):

### ISSUE THREAD

**Issue by mmikkel** [mmikkel/Reasons-Craft3#10]
Experiencing something similar when trying to add Reasons to an existing site. Package installs correctly, but then all pages display the error (presumably due to config rebuild as `project-config/rebuild` also throws same error):

```
error: SQLSTATE[42703]: Undefined column: 7 ERROR:  column reasons.fieldlayoutid does not exist
LINE 3: ...fieldlayouts" "fieldlayouts" ON fieldlayouts.id = reasons.fi...
                                                             ^
HINT:  Perhaps you meant to reference the column "reasons.fieldLayoutId".
The SQL being executed was: SELECT "reasons"."uid", "reasons"."conditionals", "fieldlayouts"."uid" AS "fieldLayoutUid"
FROM "reasons" "reasons"
INNER JOIN "fieldlayouts" "fieldlayouts" ON fieldlayouts.id = reasons.fieldLayoutId
```

Can send through a DB dump and the `config/project` directory if needed 👍

Craft: Pro 3.5.14
Reasons: 2.2.2
DB Engine: PostgreSQL 12.1

_Originally posted by @slavanossar in https://github.com/mmikkel/Reasons-Craft3/issues/9#issuecomment-720216068_

**Comment by mmikkel** [comment]
Hi @slavanossar, thanks for reporting. Not sure if this is project config related problem per se, or a Postgres issue. I'll look into it ASAP.

**Comment by slavanossar** [comment]
@mmikkel Thanks for looking into it.

So I did a bit more digging after you mentioned it might be a Postgres issue and came across [this SO answer](https://stackoverflow.com/a/41591457/4418436).

I checked the stack trace and this was the line that was causing the error:
https://github.com/mmikkel/Reasons-Craft3/blob/c55bfc7cf67b41af826aa8649b65b1be4455713b/src/services/ReasonsService.php#L361

Changing it to use `"fieldLayoutId"` fixes the error, and I created a few conditional field layouts as a test and everything seems to be working correctly.
